USER_MANUAL.md

```diff
@@ -0,0 +1,34 @@
+# SpatialOS GDK for Unreal Test Gyms Project 
+
+#### What is a gym
+* A gym is a level containing a minimal number of actors, with simple behaviours, necessary to demonstrate some peice of functionality in the Spatial GDK for Unreal.
+* We use gyms to help quickly visualize if some feature is working.
+
+#### Project layout
+```
+Content
+|
+|__Actors
+|
+|__Characters
+|
+|__GameModes
+|
+|__Maps
+|
+|__Meshes
+|
+|__Spatial
+```
+
+New gym levels should be stored in `Content\Maps`.
+New actors to be used in gyms should be stored in `Contetnt\Actors`.
+
+#### How to add a gym 
+* Create a new level, with a descriptive name, and store it in the `Content\Maps` directory;
+  * consider copying the `EmptyGym` level in `Content\Maps` as a starting point.
+* Consider adding a text object containing the gym name to the level;
+  * this aids indentification from screen shots etc.
+* Populate the new level with actors needed to demonstrate the desired functionality;
+  * prefer to use existing actors if possible,
+  * if new actors are needed then store them in `Content\Actors`.
```

Comment:
Do we want to have a guideline that all new gyms should have a README or other doc that documents the functionality tested in the gym?

Otherwise, LGTM.

Reply:
That sounds like something we should do. Will add a section underneath this and add the handover gym and the empty gym to it.